src/main/java/io/appform/dropwizard/actors/base/Handler.java

```diff
@@ -82,4 +101,43 @@ public void handleDelivery(String consumerTag, Envelope envelope,
     private MessageMetadata messageProperties(final Envelope envelope) {
         return new MessageMetadata(envelope.isRedeliver());
     }
+
+    public static Span buildChildSpan(AMQP.BasicProperties props, Tracer tracer) {
+        Tracer.SpanBuilder spanBuilder = tracer.buildSpan("receive")
```

Comment:
add null checks and wrap around try catch

Reply:
moved all the tracing related logic in TracingHandler and put everything behind try-catch